### ISSUE TITLE
Blank Page for Spools and Filaments

### ISSUE THREAD

**Issue by blackadar** [Donkie/Spoolman#426]
**Describe the bug**
I've noticed that Spoolman can get into a state where clicking on Spools or Filaments from the home page just displays a blank page with browser errors in console.

**To Reproduce**
1. Open Spoolman
2. Click Spools
3. Page may be blank

**Expected behavior**
Page should load

**Screenshots**
![image](https://github.com/user-attachments/assets/dd25e66f-1780-4e19-9313-9171cb2e62fa)
![image](https://github.com/user-attachments/assets/abf29233-c9e6-478e-b8b1-4758cd030c3b)

**Spoolman Host (please complete the following information):**
 - Installation type: Docker
 - Installed version: Version 0.18.1 (306dbe4)
 - OS and Distro: Ubuntu
 - Database type: SQLite

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Browser: Chrome and Edge on same computer

**Additional context**
Docker container logs show no problems

**Comment by blackadar** [comment]
Also notable: I have this exposed to the internet behind Cloudflare Access under a domain name. When I access the service locally through the same browser that has the blank page, the Spools and Filaments page loads just fine.

Thinking this could either be cache related, or an interaction with Argo tunnels.

**Comment by haynesdavid** [comment]
Same issue, wont let me add, page goes blank.

**Comment by filcuk** [comment]
Same, also proxied through Cloudflare.  
Worked fine on desktop, am on mobile now (Android, Firefox).  
Screen seems to load for a split second before going white.  
  
Will add more info when I can test from a desktop again.

**Comment by haynesdavid** [comment]
does the same on desktop too, i get the screen for 2 secconds also, before going white.

**Comment by mwmason** [comment]
Non-proxied, local network - get a blank page on Spool>Clone.  I can Create a spool, from both Spools and Filaments.  Filament>Clone works.

**Comment by haynesdavid** [comment]
Mine has resolved itself, not sure if an update has been push out somewhere, works across all devices now.

**Comment by Donkie** [comment]
Very strange issue. Would be interesting to see more console errors if it appears again.

It looks like one of the javascript files failed to download fully, and then it was stored in your browser's cache causing it to keep happening. Until your browser decided to download it again whereupon it now succeeds.

Maybe it has something to do with cloudflare?

**Comment by blackadar** [comment]
Happened again for me today, here's the console errors:

![image](https://github.com/user-attachments/assets/669a155c-08d0-4fc2-a351-6ea2fe2d0278)

Let me know if anything else would be helpful :)

**Comment by Donkie** [comment]
The old client is no longer served by default as of 0.26.0. Closing as stale, please open a fresh issue if the new client ever renders blank.
